delft3dcontainermanager/tasks.py

```diff
@@ -52,34 +60,40 @@ def do_docker_create(self, image):
     return container_id
 
 
-@shared_task(bind=True)
+@shared_task(bind=True, throws=(HTTPError))
```

Comment:
Task can also throw 'docker.errors.NotFound' when container 'container_id' is not created.